### PR TITLE
Docs: Update docs on CamelCase

### PR DESCRIPTION
**Pull request by twMat**

### CODE REVIEW

**Comment by Marxsal** [comment]
I guess we think alike -- I just added a PR with a camelcase how-to.

**Comment by twMat** [comment]
:-)
After my PR I noted there is the tiddler [Linking in Wikitext](http://tiddlywiki.com/#Linking%20in%20WikiText) though. Just maybe all things concerning links should be dealt with in there?

**Comment by Marxsal** [comment]
When we've agreed on a title (just changed BTW) for the How-To, then we can just add a link to it in the Linking in Wikitext. That way the information is easily available no matter how someone approaches.

**Comment by Jermolene** [comment]
Hi @twMat thanks for this. As per the discussion in #2687, I think it's better to add the instructions as a separate tiddler, rather than including them in a "concepts" tiddler.

File: editions/tw5.com/tiddlers/definitions/CamelCase.tid [hunk 1]
```diff
@@ -4,3 +4,5 @@ tags: Definitions
 title: CamelCase
 
 CamelCase is formed by taking a phrase, capitalising the initial letter of each word, and smashing the words together to form a portmanteau word. Most wikis use CamelCase to signal phrases that should automatically become links.
+
+The automatic conversion to a link can be disabled [[here|$:/core/ui/ControlPanel/Settings/CamelCase]].
```
Comment (twMat) [inline editions/tw5.com/tiddlers/definitions/CamelCase.tid]: I think this pull request was justifiably dismissed all together given #2687 so it is not a matter.
That said, I agree it should have been [[Controlpanel|$:/core/ui/ControlPanel/Settings/CamelCase ]].

Comment (Jermolene) [inline editions/tw5.com/tiddlers/definitions/CamelCase.tid]: Sorry @twMat! We don't often have the luxury of choosing between two PRs for the same issue;  I'm just tickled that you and @Marxsal were both so responsive. Much appreciated from you both.

**Pull request closed without merging**